resources/searches/Application-Exploration-Table.json

```diff
@@ -0,0 +1,23 @@
+{
+   "sort": [
+      "TimeUpdated", 
+      "desc"
+   ], 
+   "hits": 0, 
+   "description": "", 
+   "title": "Application Exploration Table", 
+   "version": 1, 
+   "kibanaSavedObjectMeta": {
+      "searchSourceJSON": "{\"index\":\"[network_]YYYY_MM_DD\",\"highlight\":{\"pre_tags\":[\"@kibana-highlighted-field@\"],\"post_tags\":[\"@/kibana-highlighted-field@\"],\"fields\":{\"*\":{}},\"fragment_size\":2147483647},\"filter\":[],\"query\":{\"query_string\":{\"query\":\"*\",\"analyze_wildcard\":true}}}"
+   }, 
+   "columns": [
+      "SrcIP", 
+      "DestIP", 
+      "Application", 
+      "ApplicationFamily", 
+      "ApplicationTags", 
+      "TotalBytesDelta", 
+      "PacketsDelta", 
```

Comment:
Depending on how a search table is created it may or may not allow `export`. I'm not exactly clear on why it sometimes allow that and for other search tables it doesn't 

This search table does *not* allow export of the data which should be addressed if possible.